### PR TITLE
DOCS-2422 Automate Remediated Threats guide

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds a new guide for automating remediated threats with security monitoring

### Motivation
Jira requeat

### Preview

https://docs-staging.datadoghq.com/sarina/sec-mon-webhooks/security_platform/guide/automate-the-remediation-of-detected-threats/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/guide/_index.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ private: true
 ---
 
 {{< whatsnext desc="General Guides:" >}}
+    {{< nextlink href="/security_monitoring/guide/automate-the-remediation-of-detected-threats" >}}Automate the remediation of detected threats with Security Monitoring{{< /nextlink >}}
```
Comment (sarina-dd) [inline content/en/security_platform/guide/_index.md:10]:
```suggestion
    {{< nextlink href="/security_platform/guide/automate-the-remediation-of-detected-threats" >}}Automate the remediation of detected threats with Security Monitoring{{< /nextlink >}}
```

File: content/en/security_platform/guide/_index.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ private: true
...
     {{< nextlink href="security_monitoring/guide/monitor-authentication-logs-for-security-threats" >}}Monitor Authentication Logs for Security Threats{{< /nextlink >}}
```
Comment (sarina-dd) [inline content/en/security_platform/guide/_index.md:10]: Update these links

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Security Monitoring allows you to set Detection Rules that trigger auto-remediation workflows. With Datadog’s [webhook integration][1], set up webhooks to deliver payloads to the services you want to automate whenever a [Detection Rule][2] is broken. Every webhook payload contains information about the triggering event and a custom message that can be used to initiate services downstream. Automate commands for any service that has a webhook URL.
+
+Choose a security scenario below to get started.
+
+## Delete misconfigured security groups
+
+In a cloud environment, it’s important to delete a misconfigured resource as soon as it is created. To do this, configure the [webhook integration][1] to send a [webhook][1] to your cloud provider’s API management service.
+
+{{< img src="security_platform/guides/automate-the-remediation-of-detected-threats/automation-diagram.png" alt="A diagram for a webhook sent to a cloud provider's API" >}}
```
Comment (sarina-dd) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:16]: Check imgs as they aren't rendering

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        "ip": [
+          "1.2.3.4"
+        ]
+      }
+    }
+  }
+}
+{{< /code-block >}}
+
+Datadog generates the Security Signal, which details the offense as well as the suspicious user’s information, such as their IP address and user ID, and the webhook payload will send remediation instructions to a service to disable the user's credentials.
```
Comment (sarina-dd) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:81]: Add concluding section + further reading

**Review by sarina-dd — COMMENTED** [review]
Noted updates that need to be made

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
```
Comment (sarina-dd) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:86]:
```suggestion
Datadog generates the Security Signal, which details the offense as well as the suspicious user’s information, such as their IP address and user ID, and the webhook payload sends remediation instructions to a service to disable the user's credentials.
```

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+{{< img src="security_platform/security_monitoring/guide/automate-the-remediation-of-detected-threats/new-term-rule.png" alt="A new term-based detection rule" >}}
+
+Then, set up a [webhook][2] that will send a payload to your cloud’s identity and access management (IAM) service to ban the unknown IP when this rule is triggered.
+
+{{< img src="security_platform/security_monitoring/guide/automate-the-remediation-of-detected-threats/webhook-ip.png" alt="A new webhook that bans an unknown IP address" >}}
+
+The following example illustrates what the relevant webhook payload could look like when a security signal is produced by Datadog:
+
+{{< code-block lang="json" filename="webhook-payload.json" >}}
```
Comment (sarina-dd) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:41]: @jmassey-datadog Hey there! Need verification / review here that these code blocks are good to go / any add't recommendations you may have for the guide.

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+further_reading:
+- link: "/security_platform/explorer/"
+  tag: "Documentation"
+  text: "Start investigating signals in the Signals Explorer"
+
+---
+
+## Overview
+
+[Security Monitoring][1] allows you to set Detection Rules that trigger auto-remediation workflows. With Datadog’s [webhook integration][2], set up webhooks to deliver payloads to the services you want to automate whenever a [Detection Rule][3] is broken. Every webhook payload contains information about the triggering event and a custom message that can be used to initiate services downstream. Automate commands for any service that has a webhook URL.
```
Comment (jmassey-datadog) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:13]: > is broken

this should say something a bout when a detection rule is triggered. not when something is broken.

> be used to initiate services downstream

I wouldn't say "initiate". I would say something like "can be passed to services downstream" or "can be used by downstream services".

**Review by jmassey-datadog — COMMENTED** [review]
started review - will come back for more later

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md (hunk 7 above)
Comment (sarina-dd) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:13]:
```suggestion
[Security Monitoring][1] allows you to set Detection Rules that trigger auto-remediation workflows. With Datadog’s [webhook integration][2], set up webhooks to deliver payloads to the services you want to automate whenever a [Detection Rule][3] is triggered. Every webhook payload contains information about the triggering event and a custom message that can be used to initiate services downstream. Automate commands for any service that has a webhook URL.
```

Comment (sarina-dd) [reply content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:13]: +1 on adding more detail, any suggestions? Following what the blog says, and not quite sure what we mean by commands here -- actual commands or the automated processes that occur within a service?

Comment (sarina-dd) [reply content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md]: @jmassey-datadog Bumping for further review 😸

Comment (jmassey-datadog) [reply content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md]: these "commands" are actually talking about the scenarios below. So maybe a single sentence summarizing the 3 different scenarios?  Or something like

> Security Orchestration and Automation Response tools accept incoming HTTP requests and these webhooks will initiate any workflow you have defined.

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+further_reading:
+- link: "/security_platform/explorer/"
+  tag: "Documentation"
+  text: "Start investigating signals in the Signals Explorer"
+
+---
+
+## Overview
+
+[Security Monitoring][1] allows you to set Detection Rules that trigger auto-remediation workflows. With Datadog’s [webhook integration][2], set up webhooks to deliver payloads to the services you want to automate whenever a [Detection Rule][3] is triggered. Every webhook payload contains information about the triggering event and a custom message that can be used to initiate services downstream. Automate commands for any service that has a webhook URL. Security orchestration and automation response tools accept incoming HTTP requests and these webhooks will initiate any workflow you have defined.
```
Comment (jtappa) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:13]:
```suggestion
[Security Monitoring][1] allows you to set Detection Rules that trigger auto-remediation workflows. With Datadog’s [webhook integration][2], set up webhooks to deliver payloads to the services you want to automate whenever a [Detection Rule][3] is triggered. Every webhook payload contains information about the triggering event and a custom message that can be used to initiate services downstream. Automate commands for any service that has a webhook URL. Security orchestration and automation response tools accept incoming HTTP requests and these webhooks initiate any workflow you have defined.
```

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+Choose a security scenario below to begin automating remediation.
+
+## Delete misconfigured security groups
+
+In a cloud environment, it’s important to delete a misconfigured resource as soon as it is created. In this scenario, you can configure a [webhook integration][2] to send a [webhook][2] to your cloud provider’s API management service.
+
+{{< img src="security_platform/security_monitoring/guide/automate-the-remediation-of-detected-threats/automation-diagram.png" alt="A diagram for a webhook sent to a cloud provider's API" >}}
+
+Once configured, if an AWS user creates a poorly configured resource (for example, an overly permissive security group, user role, etc.) within your AWS environment, Datadog Log Management will ingest the related log, which triggers a security group–based Detection Rule. This process will automatically send the webhook’s JSON payload to the designated AWS API Gateway URL, which in turn activates a AWS Lambda function that automatically deletes the offending resource.
```
Comment (jtappa) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:23]:
```suggestion
Once configured, if an AWS user creates a poorly configured resource (for example, an overly permissive security group, or user role) within your AWS environment, Datadog Log Management ingests the related log, which triggers a security group–based Detection Rule. This process automatically sends the webhook’s JSON payload to the designated AWS API Gateway URL, which in turn activates an AWS Lambda function that automatically deletes the offending resource.
```

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+A sign-in from an unrecognized IP address might represent an attacker manipulating a trusted user’s credentials, with which they can then access your data and gain persistence in your environment.
+
+To combat this type of attack, you can use the [new term detection method][4], which analyzes your account’s historical data over a chosen period of time and alerts on previously unseen values in your cloud logs.
+
+First, set up a [new term–based Detection Rule][5].
+
+{{< img src="security_platform/security_monitoring/guide/automate-the-remediation-of-detected-threats/new-term-rule.png" alt="A new term-based detection rule" >}}
+
+Then, set up a [webhook][2] that will send a payload to your cloud’s identity and access management (IAM) service to ban the unknown IP when this rule is triggered.
```
Comment (jtappa) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:35]:
```suggestion
Then, set up a [webhook][2] that sends a payload to your cloud’s identity and access management (IAM) service to ban the unknown IP when this rule is triggered.
```

File: content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+        ]
+      }
+    }
+  }
+}
+{{< /code-block >}}
+
+## Application abuse and fraud
+
+With Datadog Security Monitoring, uncover patterns of [abuse or fraud][6] across your application. For example, set up a [Detection Rule][7] that is triggered when a user repeatedly attempts to purchase something in your application with invalid credit card details. Then, set up a webhook that sends a payload with remediation instructions to a service that will disable the user's credentials.
```
Comment (jtappa) [inline content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md:59]:
```suggestion
With Datadog Security Monitoring, you can uncover patterns of [abuse or fraud][6] across your application. For example, set up a [Detection Rule][7] that is triggered when a user repeatedly attempts to purchase something in your application with invalid credit card details. Then, set up a webhook that sends a payload with remediation instructions to a service that will disable the user's credentials.
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by jtappa — APPROVED** [review]

**Pull request merged**